### ISSUE TITLE
[P2][Sourcing][Accessibility][ATL Testing][a11y help]multi-input ariaLabel not working

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10393]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
multi-select ariaLabel not working and hence not read by screenreader
Sometime "Multi input value" is being read if ariaLabel is undefined , but when user passes the value, the aria-label property on input field of multi-input becomes empty

Expectation: 
When user passes a value to "ariaLabel" of fd-multi-input or fdp-multi-input, the value should be read by the screenreader when focus is on input field 

#### If this is a bug, please provide steps for reproducing it.
able to replicate on https://sap.github.io/fundamental-ngx/#/core/multi-input#simple

**Comment by fkolar** [comment]
@I543348, you mean the aria-label binding on MultiInput is not added to the Input html tag right? Instead there is some default value added .

**Comment by I543348** [comment]
Hi @fkolar , this is the same issue we discussed some days back during meeting with ATL.
The value that is sent from the developer for ariaLabel is not getting reflected in the DOM and comes as empty

**Comment by fkolar** [comment]
Yes, just little clarification for F-NGX team so they know exactly, always provide more than less information .. so in this case would good to have. 

As you can see on the picture ariaLabel set HELLOHELLO is not set on the input field, there is just some default value instead. 

<img width="1215" alt="Screenshot 2023-08-30 at 8 40 43" src="https://github.com/SAP/fundamental-ngx/assets/17149942/ed9fe9ea-60ac-4359-aced-9adc8c60a287">

**Comment by I543348** [comment]
Unable to verify the issue as deploy preview URL stackblitz is not working 
i dont see app.module.ts file and neither any control is coming up on the screen.
pls see screenshot.
<img width="1792" alt="Screenshot 2023-09-11 at 5 37 37 PM" src="https://github.com/SAP/fundamental-ngx/assets/88182358/fba75e4a-0006-46fc-ba2b-4fceede36adf">

**Comment by sagar7259** [comment]
Hello Team,

Now the label and UI type of a 'Multi value input' is read correctly. hence This ticket can be closed.

Regards,
Sagar
